keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -87,7 +87,7 @@ def remove_strings_from_inputs(tensor, string_to_remove):
     return result
 
 
-def split_strings_for_bpe(inputs):
+def split_strings_for_bpe(inputs, add_prefix_space):
```

Comment:
Will this work 🤔? We are passing this argument in this function, but not actually doing anything with it inside the function.

I was thinking of doing something along these lines in the [`tokenize()`](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/tokenizers/byte_pair_tokenizer.py#L450) function:
```
    def tokenize(self, inputs):
        if not isinstance(inputs, (tf.Tensor, tf.RaggedTensor)):
            inputs = tf.convert_to_tensor(inputs)
        if self.add_prefix_space:
            inputs = tf.strings.join([" ", inputs])
        ...
```

Trial:
```
>>> import tensorflow as tf
>>> inputs = tf.constant(["add space at the beginning of every string", "we can use tf.strings.join(...)"])
>>> inputs
<tf.Tensor: shape=(2,), dtype=string, numpy=
array([b' add space at the beginning of every string',
       b' we can use tf.strings.join(...)'], dtype=object)>
```

CC: @mattdangerw